tests/src/JIT/HardwareIntrinsics/X86/Shared/_BooleanBinaryOpTestTemplate.template

```diff
@@ -52,6 +52,12 @@ namespace JIT.HardwareIntrinsics.X86
                 // Validates passing a static member works
                 test.RunClsVarScenario();
 
+                if ({LoadIsa}.IsSupported)
```

Comment:
This, https://github.com/dotnet/coreclr/pull/24455/commits/5037ebe762cb6d1254f89fd25bc282e78d19e55a#diff-d3f963eacee64aaa85d71d455212cdb2R55, https://github.com/dotnet/coreclr/pull/24455/commits/5037ebe762cb6d1254f89fd25bc282e78d19e55a#diff-7566dd258938e7753c316937dd6df057R55, and https://github.com/dotnet/coreclr/pull/24455/commits/5037ebe762cb6d1254f89fd25bc282e78d19e55a#diff-c6ae68adf332b967d6f190c58031da65R55 were all updated to include the additional addressing mode coverage.

The other shared template, `_BinaryOpTest.template`, was ported from a template that already had that coverage.